core/src/main/java/io/undertow/server/SSLSessionInfo.java

```diff
@@ -45,24 +45,28 @@ static int calculateKeySize(String cipherSuite) {
         // Roughly ordered from most common to least common.
         if (cipherSuite == null) {
             return 0;
-        } else if (cipherSuite.contains("WITH_AES_256_")) {
+        } else if (cipherSuite.contains("AES_256_")) {
             return 256;
-        } else if (cipherSuite.contains("WITH_RC4_128_")) {
+        } else if (cipherSuite.contains("RC4_128_")) {
             return 128;
-        } else if (cipherSuite.contains("WITH_AES_128_")) {
+        } else if (cipherSuite.contains("AES_128_")) {
             return 128;
-        } else if (cipherSuite.contains("WITH_RC4_40_")) {
+        } else if (cipherSuite.contains("RC4_40_")) {
             return 40;
-        } else if (cipherSuite.contains("WITH_3DES_EDE_CBC_")) {
+        } else if (cipherSuite.contains("3DES_EDE_CBC_")) {
```

Comment:
Oh, boy this will be fun: https://www.ibm.com/docs/en/zos/2.3.0?topic=programming-cipher-suite-definitions
TLS_DH_DSS_WITH_3DES_EDE_CBC_SHA --> "168-bit Triple DES encryption with SHA-1 message authentication and fixed Diffie-Hellman key exchange signed with a DSA certificate"

But according to https://www.rfc-editor.org/rfc/rfc5246.html  --> "DES can also be
      operated in a mode [[3DES](https://www.rfc-editor.org/rfc/rfc5246.html#ref-3DES)] where three independent keys and three
      encryptions are used for each block of data; this uses 168 bits of
      key (24 bytes in the TLS key generation method) and provides the
      equivalent of 112 bits of security."

Reply:
This one actually is inaccurate in one case TLS_KRB5_EXPORT_WITH_DES_CBC_40_MD5 - which is 40